src/__test__/text-field/text-field-actions.test.js

```diff
@@ -0,0 +1,60 @@
+import expect from 'expect';
+import {
+  CLEAR_TEXT_FIELDS,
+  REMOVE_INPUT_ERROR,
+  removeAllTextFieldDataAndErrors,
+  removeInputError,
+  SET_INPUT_ERROR,
+  SET_TEXT_FIELD_DATA,
+  setInputError,
+  setTextFieldData,
+} from '../../component/bpm-text-field/text-field-actions';
+
+describe('Tests text field actions', () => {
+  it('Creates an action to update template creation data', () => {
+    const field = 'fieldName';
+    const value = 'fieldValue';
+
+    const expectedAction = {
+      type: SET_TEXT_FIELD_DATA,
+      field,
+      value,
+    };
+
+    expect(setTextFieldData(field, value))
+      .toEqual(expectedAction);
+  });
```

Comment:
we can put the `expect` and `toEqual` instructions in the same line. Otherwise this file will be really large.

Reply:
done